documentation/basics/batch.md

```diff
@@ -7,27 +7,59 @@ weight: 6
+
+The next step is the actual batch processing itself.
+
+<a href="screenshots/batch_processing_02.png" data-toggle="lightbox"><img src="screenshots/batch_processing_02.png" class="img-responsive" /></a>
+
+After clicking on the **Process all files** button, ilastik begins batch processing all images, and writes the resulting classification result to the specified output files.
+With default export settings, the output files are stored as hdf5 files in the same directory where the input file is located.
+Their file names will be the same as the original files, plus one or two words indicating the export source (e.g. "Probabilities") and the file extension `.h5`.
+
+The exported `.h5` files, contain the resulting prediction as a multidimensional dataset inside the file.
+Further details on this versatile file format (that is easily accessible from Matlab and Python) can be found [here](http://docs.h5py.org/).
 
-The first step in batch prediction is the selection of the input files. This uses the same mechanism as [data selection]({{baseurl}}/documentation/basics/dataselection.html).
+### Batch Processing FAQ
 
-<a href="screenshots/batch1_zoomed.png" data-toggle="lightbox"><img src="screenshots/batch1_zoomed.png" class="img-responsive" /></a>
+#### I have multiple image stacks I want to process, is it possible to load those in batch?
+Image stacks are currently not supported in batch: Whenever your image is spread across multiple files, use the [headless mode][headless], or the [Fiji plugin][fijiplugin] to process those.
+Alternatively you could also convert your stack to a single file and use the batch processing graphical user interface in ilastik.
 
-When clicking on the **Add files** button, the user can choose between adding
-single or multiple files from the File selection dialog, or adding files matching a pattern from a directory (last option). This is especially useful when large amounts of images have to be processed.
 
-The next step is the actual batch processing itself. In the **Batch Prediction Output Locations** applet, the user can configure the output format using the **Choose Settings** button. This allow for example to save the results of the workflow as `.png` images rather than `.tiff` or `.h5` files. For novice users the default settings should be fine, however the results of some workflows such as the [density counting workflow]({{site.baseurl}}/documentation/counting/counting.html) cannot be exported in some of these formats (**hdf5 file format is supported by all workflows**).
+#### How do I specify the internal dataset in a `.hdf5`/`.z5`-file?
+It is currently not possible to specify the internal dataset with the graphical user interface, use the [headless mode][headless], or [Fiji plugin](https://github.com/ilastik/ilastik4ij#ilastik-imagej-modules) to process that data.
+Alternatively you could create `hdf5` files with only single virtual dataset each, that point to the file that has multiple ones.
```

Comment:
I don't follow how this suggestion achieves "specifying the internal dataset". Are we asking the user to wrap the batch processing output in another h5?

Reply:
yeah, maybe that's a tad too advanced... So in h5 there is the concept of [virtual datasets](). So what I'm suggesting is to create one for each of the internal paths - such h5 files would appear as if there were only a single dataset. I'll scratch that remark, and wait until such a question actually comes up (in the forum) or we devote some time on improving the ui :)